### ISSUE TITLE
Does Rove *require* ASDF in order to run a test suite? (It's not documented)

### ISSUE THREAD

**Issue by jpellegrini** [fukamachi/rove#13]
Hello,

I see that Rove has ASDF integration, which is great. But it seems that (as far as I could understand) it also _requires_ that an ASDF definition exists in order for a test suite to run (except for individual tests, which can be run wirh `run-test`, but that is not thte same thing as running the full test suite).

It would be nice to add a paragraph to the README explaining that -- or even better, to have a function that will run a test suite without the need of an ASDF system with components having being defined beforehand.

(I know most real systems will use ASDF, but sometimes a testing framework is useful when you want to test a simple/small program for which you wouldn't bother using ASDF).

**Comment by fukamachi** [comment]
Yes, currently `rove:run` is considered to be used with ASDF systems or ASDF package-inferred-system.
Perhaps, it's nice if `rove:run` allows to take a package-designator to run tests in it.
(I'll add it when I'm sure it's what you need)
